### PR TITLE
Exposing additional parameters in the infomax code to ease the comparison against eeglab infomax version

### PR DESCRIPTION
**Pull request by jmontoyam**
This pull request aims to ease the comparison between mne_python infomax code against eeglab infomax code. It exposes additional parameters in the mne_python infomax code, such that they can be set by the user who wants to compare against eeglab infomax version.

### CODE REVIEW

File: mne/preprocessing/tests/test_infomax.py [hunk 1]
```diff
@@ -114,7 +115,9 @@ def test_non_square_infomax(add_noise=False):
     m = m.T
     m = pca.fit_transform(m)
     # we need extended since input signals are sub-gaussian
-    unmixing_ = infomax(m, random_state=rng, extended=True)
+    unmixing_ = infomax(m, random_state=rng, extended=True,
+                        blowup=1e4, blowup_fac=0.5, n_small_angle=20,
+                        use_bias=True)
```
Comment (agramfort) [inline mne/preprocessing/tests/test_infomax.py]: please just avoid touching the tests. It should not be necessary.

**Comment by agramfort** [comment]
where do you want to document the parameter that one should use to get the exact same result as EEGLAB?

**Comment by jmontoyam** [comment]
@Eric89GXL , @agramfort , I run the test locally and the test pass ok, but in Travis the test doesn't pass. It gives me this error:
FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/miniconda/envs/testenv/lib/python3.4/site-packages/mne-0.10.dev0-py3.4.egg/mne/preprocessing/tests/data/eeglab_infomax_results_eeg_data.mat'

It doesn't find the test data files, but I have commited them too:
https://github.com/jmontoyam/mne-python/tree/exposing_more_infomax_parameters/mne/preprocessing/tests/data

To create the test data path, I've followed this:
https://github.com/mne-tools/mne-python/blob/master/mne/io/fiff/tests/test_raw.py#L37

This is how I load the data:
eeglab_data = sio.loadmat(base_dir + '/' + eeglab_results_file)
where base_dir is
base_dir = op.join(op.dirname(__file__), 'data')

Could you please give some advice to solve this issue?

Why can't Travis find these test data files?

Thanks in advance!

**Comment by Eric89GXL** [comment]
It is probably failing on the one that runs on the "install" instead of "source" version. You probably need to add a line to `setup.py` that makes your new data file be uploaded. BTW how big is the file? If it is large we should add it to the testing data instead of the repo.

**Comment by agramfort** [comment]
files are 4kb so we're good. We need to update manifest.in file that's it.

**Comment by jmontoyam** [comment]
Travis is happy!, thanks @agramfort!! :-)

**Comment by agramfort** [comment]
great !

**Pull request merged**